src/base.ts

```diff
@@ -42,18 +54,32 @@ export enum ChainKey {
   ONE = 'one',
   FSN = 'fsn',
   MOR = 'mor',
+  EXP = 'exp',
+  TCH = 'tch',
+  UBQ = 'ubq',
+  META = 'meta',
+  DIODE = 'diode',
+  CELO = 'celo',
+  FUSE = 'fuse',
+  TLOS = 'tlos',
+  CRO = 'cro',
+  BOB = 'boba',
 
   // Testnets
   ROP = 'rop',
   RIN = 'rin',
   GOR = 'gor',
+  KAL = 'kal',
+  SDIODE = 'sdiode',
   KOV = 'kov',
   MUM = 'mum',
   ARBT = 'arbt',
   OPTT = 'optt',
   BSCT = 'bsct',
   HECT = 'hect',
   ONET = 'onet',
+  SPARK = 'spark',
+  TLOST = 'tlost',
```

Comment:
I would like to use three letter versions of the chain Keys. Please shorten them. The Testnets can have 3 letters of the main chain and a `T` for testnet.

Reply:
got it